inlong-manager/manager-client-examples/src/test/java/org/apache/inlong/manager/client/ut/BaseTest.java

```diff
@@ -0,0 +1,170 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.inlong.manager.client.ut;
+
+import com.github.tomakehurst.wiremock.WireMockServer;
+import com.github.tomakehurst.wiremock.client.WireMock;
+import com.google.common.collect.Lists;
+import org.apache.inlong.manager.client.api.ClientConfiguration;
+import org.apache.inlong.manager.client.api.InlongClient;
+import org.apache.inlong.manager.common.auth.DefaultAuthentication;
+import org.apache.inlong.manager.common.enums.DataSeparator;
+import org.apache.inlong.manager.common.enums.FieldType;
+import org.apache.inlong.manager.common.enums.FileFormat;
+import org.apache.inlong.manager.common.enums.GlobalConstants;
+import org.apache.inlong.manager.common.pojo.group.InlongGroupInfo;
+import org.apache.inlong.manager.common.pojo.group.pulsar.InlongPulsarInfo;
+import org.apache.inlong.manager.common.pojo.sink.SinkField;
+import org.apache.inlong.manager.common.pojo.sink.hive.HiveSink;
+import org.apache.inlong.manager.common.pojo.sort.FlinkSortConf;
+import org.apache.inlong.manager.common.pojo.stream.InlongStreamInfo;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+
+public class BaseTest {
+
+    // Manager web url
+    public static final String SERVICE_URL = "127.0.0.1:8084";
+    // Inlong user && passwd
+    public static DefaultAuthentication inlongAuth = new DefaultAuthentication("admin", "inlong");
+    // Inlong group ID
+    public static final String GROUP_ID = "test_group009";
+    // Inlong stream ID
+    public static final String STREAM_ID = "test_stream009";
+    // Flink cluster url
+    public static final String FLINK_URL = "127.0.0.1";
+    // Pulsar cluster admin url
+    public static final String PULSAR_ADMIN_URL = "127.0.0.1";
+    // Pulsar cluster service url
+    public static final String PULSAR_SERVICE_URL = "127.0.0.1";
+    // Pulsar tenant
+    public static final String TENANT = "tenant";
+    // Pulsar tenant
+    public static final String NAMESPACE = "test_namespace";
+    // Pulsar topic
+    public static final String TOPIC = "test_topic";
+    public static final String IN_CHARGES = "test_inCharges,admin";
+
+    public static WireMockServer wireMockServer;
+    public static final String MANAGER_URL_PREFIX = "/api/inlong/manager";
+
+    public static InlongGroupInfo groupInfo;
+    public static InlongClient inlongClient;
+
+    /**
+     * Create inlong group info
+     */
+    public static InlongGroupInfo createGroupInfo() {
```

Comment:
Is 'static' necessary for createGroupInfo?
@BeforeAll @AfterAll method should be put above all other methods

Reply:
yes, It is initialized in static method